### PR TITLE
Add rake task that delete all the comments older than one day

### PR DESCRIPTION
**Pull request by Johnnycus**
Add rake task for heroku scheduler.
1. Install scheduler addon: `heroku addons:create scheduler:standard`.
2. Deploy task to heroku.
3. Run `heroku run rake rm_comments` for test. **It will delete all comments from project page.**
4. Open scheduler page `heroku addons:open scheduler`.
5. Set it like this: scheduler page

### CODE REVIEW

File: lib/tasks/scheduler.rake [hunk 1]
```diff
@@ -0,0 +1,5 @@
+desc "Remove all comments older than one day"
+task :rm_comments => :environment do
+  puts "Removing comments older than one day"
+  Comment.delete_all(:created_at => Time.now.midnight - 1.year..Time.now.midnight - 1.day)
```
Comment (justin808) [inline lib/tasks/scheduler.rake]: We use the ruby 1.9 has syntax `created_at:`

Comment (justin808) [inline lib/tasks/scheduler.rake]: Why does 1 year back matter?

**Comment by justin808** [comment]
We need to adjust the UI with some stylish syntax that suggests all comments older than 24 hours get deleted each day, and that markdown is supported.

File: lib/tasks/scheduler.rake (hunk 1 above)
Comment (Johnnycus) [inline lib/tasks/scheduler.rake]: 1 year is needed to delete all the previous comments, not only for the last day

**Comment by justin808** [comment]
See #62. Already done.

**Pull request closed without merging**